packages/openid4vc-client/tests/setup.ts

```diff
@@ -1,3 +1,6 @@
 import 'reflect-metadata'
+import console from 'console'
+
+global.console = console
```

Comment:
Was just used for dev or do we want to keep this in?

Reply:
You're right, removed it ;)